packages/core/admin/admin/src/pages/SettingsPage/utils/defaultRoutes.js

```diff
@@ -9,11 +8,6 @@ import WebhooksEditView from '../../Webhooks/ProtectedEditView';
 import WebhooksListView from '../../Webhooks/ProtectedListView';
 
 const defaultRoutes = [
-  {
-    Component: ApplicationInfosPage,
-    to: '/settings/application-infos',
-    exact: true,
-  },
```

Comment:
We don't need that link?

Reply:
I have created a dedicated Route for this since it is a static one.